05-knative-source-jira/README.md

```diff
@@ -0,0 +1,180 @@
+# Camel Knative JIRA Source Basic Example
+
+This example demonstrates how to get started with Camel based Knative sources by showing you some of the most important
+features that you should know before trying to develop more complex examples.
+
+You can find more information about Apache Camel and Apache Camel K on the [official Camel website](https://camel.apache.org).
+
+## Before you begin
+
+Read the general instructions in the [root README.md file](../README.md) for setting up your environment and the Kubernetes cluster before looking at this example.
+
+Make sure you've read the [installation instructions](https://camel.apache.org/camel-k/latest/installation/installation.html) for your specific
```

Comment:
that should point to the exact version number, see: https://github.com/apache/camel-k-examples/blob/master/04-aws-kinesis-source-basic/readme.didact.md#L49

Reply:
I updated knative installation links .. but this one you mentioned is the same also in  https://github.com/apache/camel-k-examples/blob/master/04-aws-kinesis-source-basic/readme.didact.md